lib/propolis/Cargo.toml

```diff
@@ -23,7 +24,8 @@ usdt = { version = "0.3.2", default-features = false }
 tokio = { version = "1", features = ["full"] }
 futures = "0.3"
 anyhow = "1"
-rfb = { git = "https://github.com/oxidecomputer/rfb", branch = "main" }
+#rfb = { git = "https://github.com/oxidecomputer/rfb", branch = "main" }
+rfb = { path = "/home/jordan/src/rfb" }
```

Comment:
I assume there's a corresponding branch/changeset which this represents?  Maybe use the git dependency to reference that branch while the PR is in review, so folks (and the CI checks) can run against it?

Reply:
good point; done in 587c21e